React/wbooks/src/redux/auth/actions.js

```diff
@@ -0,0 +1,47 @@
+import api from '../../Config/api';
+import history from '../../Config/history';
+import localStorage from '../../Services/LocalStorage';
+import UserService from '../../Services/UserService';
+
+export const authActions = {
+  logout: 'LOGOUT',
+  login: 'LOGIN',
+  loginSuccess: 'LOGINSUCCESS',
+  loginFailure: 'LOGINFAILURE',
+  signup: 'SIGNUP',
+  signupSuccess: 'SIGNUPSUCCESS',
+  signupFailure: 'SIGNUPFAILURE'
+};
+
+export default {
```

Comment:
const ActionCreators = {
  // All that content
}

export default ActionCreators